apps/webapp/components/routes/wallet/withdraw-card.tsx

```diff
@@ -89,15 +87,11 @@ export function WithdrawCard() {
 
 const coins = [
   {
-    coin: 'USDCred',
-    nertwork: 'EOS EVM'
+    coin: 'USDT',
+    nertwork: 'Arbitrum One'
```

Comment:
Same typo here from original version. Can you check this out too, please? Thanks!